about.html

```diff
@@ -0,0 +1,135 @@
+    <title>About-page</title>
+</head>
+<body>
+    <header>
+        <div id="secondary-nav-bar" class="container">
+          <div class="wrapper">
+            <a href="#"><i class="fa-brands fa-facebook"></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">Logout</a>
+          </div>
+        </div>
+        <div id="nav-bar">
+          <a href="./index.html" id="logo">WCIB<span>2022</span></a>
+          <nav id="menu">
+            <span id="menu-close-btn">×</span>
+            <a href="./index.html" class="home-link">Home</a>
+            <a href="./index.html#program">Program</a>
+            <a href="./index.html#speakers">Speakers</a>
+            <a href="./index.html#sponsors">Sponsors</a>
+            <a href="#" class="active">About</a>
+          </nav>
+          <a href="#" id="menu-toggler">
+            <i class="fa-solid fa-bars"></i>
+          </a>
+        </div>
+      </header>
+  
+      <main>
+        <div class="container">
+          <div id="hero-about" class="wrapper">
+            <img src="./assets/img/shape-set-01.png" class="shape-set" alt="" />
+            <h1>World Congress on Industrial Biotechnology ( WCIB )</h1>
+            <div class="description">
+              <p>
+                  World Congress on Industrial Biotechnology(WCIB) will be held in Caracas,Venezuela during 08th Jan-09th Jan 2022 as the Conference of WCIB .
+                   WCIB is sponsored by ConferenceFora. It aims to be one of the leading conferences for presenting novel and fundamental advances in the fields of Industrial Biotechnology. 
+                   It also serves to foster communication among researchers and practitioners working in a wide variety of scientific areas with a common interest in Industrial Biotechnology
+              </p>
+              <p>
+                  The conference provides a platform for professionals involved in Industrial Biotechnology to exchange knowledge and gain an insight into the state of the art in the current technology, 
+                  techniques and solutions in Industrial Biotechnology as they have been developed and applied in different countries. 
+                  Participants include a wide variety of stakeholders from research and academia, to industrial sectors as well as government organizations.
+              </p>
+            </div>
+  
+            <p>
+              Do you have any question? Feel free to contact us: <br />
+              <a href="biotech@gmail.com">biotech@gmail.com</a>
+            </p>
+          </div>
+        </div>
+  
+        <section class="standout container">
+          <div class="wrapper">
+            
+            <div class="info">
+                <h2 class="title">We are eager to disscus about science.</h2>
+              <p>
+                  The conference provides a platform for professionals involved in Industrial 
```

Comment:
- [ ] Kindly add this short line bar after the About Page h2 info title and don't forget to add more space between the title and the description paragram :sunglasses::nerd_face:
- [ ] You can consider changing the color of your h2 title 🎨 🖌️ 
-  I know you can do it :muscle::skin-tone-2: 

- Your Design:

![about11](https://user-images.githubusercontent.com/79058364/170040773-e87049e6-0530-4a79-bf63-358ae6316fe9.png)


- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![about1](https://user-images.githubusercontent.com/79058364/170040740-3d24c661-041e-40a0-919a-1d519a258e70.png)